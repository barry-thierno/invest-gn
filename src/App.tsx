import Header from "layouts/Header";
import { Search } from "screens/Home/Home";
import Overview from "screens/Home/Overview/Overview";
import styled from "styled-components";

const Container = styled.div`
  text-align: center;
`;
const OverviewContainer = styled.div`
  display: flex;
  justify-content: center;
  flex: 1;
`;

function App() {
  return (
    <Container>
      <Header />
      <div className="Container">
        <Search />
      </div>
      <OverviewContainer>
        <Overview />
      </OverviewContainer>
    </Container>
  );
}

export default App;
