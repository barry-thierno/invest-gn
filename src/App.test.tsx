import { render, screen } from "@testing-library/react";
import App from "./App";

test("make shapshot of the component", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});

test("renders application header", () => {
  render(<App />);
  const linkElement = screen.getByRole("heading", { name: /invest guinea/i });
  expect(linkElement).toBeInTheDocument();
});
