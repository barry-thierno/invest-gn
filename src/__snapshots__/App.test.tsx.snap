// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`make shapshot of the component 1`] = `
<div>
  <div
    class="sc-pVTFL gSggZw"
  >
    <header
      class="sc-bdvvtL hYwHED"
    >
      <nav
        class="sc-gsDKAQ bjecW"
      >
        <h1>
          Invest Guinea
        </h1>
        <a
          class="sc-dkPtRN dDWeBq"
          href=""
        >
          Qui sommes nous
        </a>
        <a
          class="sc-dkPtRN dDWeBq"
          href=""
        >
          Sugg√©rer
        </a>
        <a
          class="sc-dkPtRN dDWeBq"
          href=""
        >
          Nous soutenir
        </a>
      </nav>
    </header>
    <div
      class="Container"
    >
      <div>
        <form>
          <div>
            <input
              class="search-input"
              id="search-input-id"
              name="pokemonName"
              placeholder="Rechercher un produit..."
              value=""
            />
            <button
              disabled=""
              type="submit"
            >
              Rechercher
            </button>
          </div>
        </form>
        <div
          class="search-result"
        >
          <div
            class="product-data-view-container"
          >
            <div
              class="product-label"
            />
            <div
              class="product-price"
            >
              0
               
              XXX
              / 
              XXX
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-jrQzAO ctwoVF"
    >
      <div>
        Chargement en cours ...
      </div>
    </div>
  </div>
</div>
`;
